gce_vars/zone

```diff
@@ -1,5 +1,7 @@
 ---
 # Variables defined for us-central1-a
-names_zonea: myinstance1,myinstance3
-zonea: us-central1-a
+name_zonea: myinstance1
+name_zoneb: myinstance2 
+zone: us-central1-a
+region: us-central1
 lb_membersa: us-central1-a/myinstance1,us-central1-a/myinstance3
```

Comment:
IIRC, there were two machines per zone and now you've gone down to one per zone. The `lb_members[ab]` variable still references four collectively.